projects/fsastorefrontstyles/scss/base/building-blocks.scss

```diff
@@ -237,3 +237,20 @@
     border-right: 7px solid transparent;
   }
 }
+
+.currency {
+  .currencyCode {
+    border: 1px solid #d3d6db;
+    border-right: 0;
+    border-top-left-radius: 0.25rem;
+    border-bottom-left-radius: 0.25rem;
+    height: 48px;
+    padding: 0 10px;
+    background-color: var(--cx-color-background);
+    color: #495057;
+  }
+  .form-control {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+  }
```

Comment:
No need for this override. It would be enough to set `margin-right` of the `span.currency` element to some negative values (-2px or -3px), with z-index of 1 (should be enough).

Reply:
I tried that solution, but I don't like the how it looks like when input is focused or invalid:

![Screenshot 2021-08-25 at 11 30 23](https://user-images.githubusercontent.com/75082466/130766392-a7d39bc5-15ec-4e8e-a28e-a0fd88cebc68.png)
![Screenshot 2021-08-25 at 11 30 13](https://user-images.githubusercontent.com/75082466/130766413-bb1bb2a2-617f-49c7-83ad-67c18183e381.png)